docs/design_decisions.md

```diff
@@ -147,6 +180,9 @@ Historie wordt aflopend gesorteerd op datum geldigheid (datumVan).
 ### DD4.2 Bij historie wordt alleen de actuele situatie van inOnderzoek getoond
 Binnen de historie-endpoints wordt alleen de actuele situatie met betrekking tot "in Onderzoek" getoond. Er wordt geen historie getoond van de onderzoeken die in het verleden hebben plaatsgevonden.  
 
+### DD4.3 Gebruik standaard queryparameters voor datums bij historisch opvragen 
+Als queryparameters voor het historisch opvragen gebruiken we "peildatum", "datumvan" en "datumtotenmet"
```

Comment:
NL API strategie heeft inmiddels gezegd dat we camelCase gebruiken voor parameters en dat hebben we overgenomen (nog niet doorgevoerd in oudere API's zoals BRP). Dus datumVan en datumTotEnMet

Reply:
Is aangepast.